test/env/create_k8s_cluster.sh

```diff
@@ -45,6 +45,7 @@ echo "$kindConfig" | kind create cluster --name "${kind_cluster_name}" --config=
 cmd="echo ${insecure_registry_address} registry >> /etc/hosts"
 for node in $(kind get nodes --name "${kind_cluster_name}"); do
   docker exec "${node}" sh -c "${cmd}"
+  docker exec "${node}" sh -c "systemctl restart containerd"
```

Comment:
is that really necessary, it happend only one for you that you had and random issue with containerd